app/liquid/views/layouts/generic-survey.liquid

```diff
@@ -0,0 +1,10 @@
+{% comment %} Description: A survey page with no branding. {% endcomment %}
+{% comment %} Experimental: true {% endcomment %}
```

Comment:
We still use comments, but because this one is experimental (ie, won't show up at all) it doesn't make sense to flag it primary or secondary. This is just for the open-source aspect - the one we use is in sou-assets, and it has `{% comment %} Primary layout: true {% endcomment %}`